src/main/java/net/sacredlabyrinth/phaed/simpleclans/managers/SettingsManager.java

```diff
@@ -84,6 +85,11 @@ public boolean is(ConfigField field) {
         return config.getBoolean(field.path, (Boolean) field.defaultValue);
     }
 
+    public boolean isIgnored(ProtectionManager.Action action, Block block) {
+       return is(WAR_LISTENERS_INVERT_IGNORED_LIST) ? !getIgnoredList(action).contains(block.getType().name())
+        : getIgnoredList(action).contains(block.getType().name());
+    }
+
```

Comment:
Thanks! 
Now everything looks good, unless _one small moment_.

Perhaps, you noticed that war and land features are the part of **one mechanism**,
`ignored-list` will affect **on both**. Consequently, it means `inverted-ignore-list` will prevent clan members to share their lands.

To avoid this situation, I can suggest to make `ignored-list` separated and each action could have it's own `invert` option. Something like:
```yaml
war-and-protection:
  listeners:
    ignored-lists:
      war:
          PLACE:
            invert: true
            blocks:
            - "PLAYER_HEAD"
      land:
          BREAK:
            invert: false
            blocks:
            - "PLAYER_HEAD"
```
Then it will require to migrate old config format to a new one.
You can look at the example of migration [here](https://github.com/RoinujNosde/SimpleClans/blob/master/src/main/java/net/sacredlabyrinth/phaed/simpleclans/migrations/BbMigration.java#L13).

If you have other ideas, or if you have problems with the implementation, feel free to ask about it.